migrations/scripts/lts-to-lts/0.23-to-0.25/playbooks/tasks/upgrade.yaml

```diff
@@ -7,9 +7,10 @@
 - name: Set up variables
   set_fact:
     backup_dir: "/astronomer-backups"
-    timestamp: "{{ lookup('pipe','date +%Y-%m-%d-%H-%M-%S') }}"
-    upgrade_to_version: "0.25"
-    upgrade_to_version_airflow: "0.19"
+    timestamp: "{{ lookup('pipe', 'date +%Y-%m-%d-%H-%M-%S') }}"
+    upgrade_to_version: "0.25.6"  # this must be a complete semver
```

Comment:
Why does this have to be a complete server? We have left the upgrade to version to 0.23 in the 0.16 to 0.23 version. This forces us to upgrade the upgraded to a new server version on every new release. If that is a thing we want then it works but wanted that to be acknowledged as work done on every patch release.

Reply:
@rbankston the two ways we've done it in the past were hacks that don't work in all cases.

more details:
* Here's where we define the major/minor version: https://github.com/astronomer/astronomer/blob/7d6fdb3a80/migrations/scripts/lts-to-lts/0.23-to-0.25/playbooks/tasks/upgrade.yaml#L11
* Here's the hack that finds the latest minor version: https://github.com/astronomer/astronomer/blob/7d6fdb3a80/migrations/scripts/lts-to-lts/0.23-to-0.25/playbooks/tasks/upgrade.yaml#L245
* Here's the commit that removed the old way: https://github.com/astronomer/astronomer/commit/d6deb7adefbfbb092b7d9f0af31aff43bb36389f#diff-efbe5156f62457226feb2e032b086805cd647894d38bbe907945340361df5d52

Reply:
More details and examples of the problem https://github.com/astronomer/issues/issues/3444

Reply:
Oh, it looks like they now support version constraints. This was changed 6 months ago. https://github.com/helm/helm/commit/171321bb6cb51198bfb63cee5f82cbfea029409a

Reply:
Fixed by passing the variable straight through now that helm 3.6 supports ranges and no longer requires an exact version match.

Reply:
Do we still need the exact patch version 0.25.6 then?